src/grbindex/grbindex.f

```diff
@@ -197,51 +197,51 @@ SUBROUTINE WRGI1H(LUGI,NLEN,NNUM,CGB)
 #else
       CHEAD(1)(56:70)=HOSTNAM(hostname)
 #endif
-C     print*,' CHEAD(1)(56:70) = ', CHEAD(1)(56:70)
+!     print*,' CHEAD(1)(56:70) = ', CHEAD(1)(56:70)
       CHEAD(1)(72:80)='grbindex '
       CHEAD(1)(81:81)=CHAR(10)
-C - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-C  FILL SECOND 81-BYTE HEADER
+! - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+!  FILL SECOND 81-BYTE HEADER
       CHEAD(2)='IX1FORM:'
       WRITE(CHEAD(2)(9:38),'(3I10)') 162,NLEN,NNUM
       CHEAD(2)(41:80)=CGB(NCGB1:NCGB)
       CHEAD(2)(81:81)=CHAR(10)
-C - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-C  WRITE HEADERS AT BEGINNING OF INDEX FILE
+! - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+!  WRITE HEADERS AT BEGINNING OF INDEX FILE
       CALL BAWRITE(LUGI,0,162,KW,CHEAD)
-C - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+! - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       RETURN
       END
-C-----------------------------------------------------------------------
+!-----------------------------------------------------------------------
       FUNCTION NCBASE(C,N)
-C$$$  SUBPROGRAM DOCUMENTATION BLOCK
-C
-C SUBPROGRAM: NCBASE         LOCATE BASENAME OF A FILE
-C   PRGMMR: IREDELL          ORG: W/NMC23     DATE: 93-11-22
-C
-C ABSTRACT: THIS SUBPROGRAM LOCATES THE CHARACTER NUMBER AFTER THE LAST 
-C   IN A CHARACTER STRING.  FOR UNIX FILENAMES, THE CHARACTER NUMBER
-C   RETURNED MARKS THE BEGINNING OF THE BASENAME OF THE FILE.
-C
-C PROGRAM HISTORY LOG:
-C   93-11-22  IREDELL
-C
-C USAGE:     ...=NCBASE(C,N)
-C   INPUT ARGUMENTS:
-C     C            CHARACTER STRING TO SEARCH
-C     N            INTEGER LENGTH OF STRING
-C
-C ATTRIBUTES:
-C   LANGUAGE: CRAY FORTRAN
-C
-C$$$
+!$$$  SUBPROGRAM DOCUMENTATION BLOCK
+!
+! SUBPROGRAM: NCBASE         LOCATE BASENAME OF A FILE
+!   PRGMMR: IREDELL          ORG: W/NMC23     DATE: 93-11-22
+!
+! ABSTRACT: THIS SUBPROGRAM LOCATES THE CHARACTER NUMBER AFTER THE LAST 
+!   IN A CHARACTER STRING.  FOR UNIX FILENAMES, THE CHARACTER NUMBER
+!   RETURNED MARKS THE BEGINNING OF THE BASENAME OF THE FILE.
+!
+! PROGRAM HISTORY LOG:
+!   93-11-22  IREDELL
```

Comment:
We've typically removed this if there's only the initial author.

Reply:
Fixed.